### PR TITLE
added stablecoin action plan

### PR DESCRIPTION
**Pull request by hexonaut**

### CODE REVIEW

File: governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ * TUSD-A
+ * PAXUSD-A
+
+ This executive will then add the following vault types with the same settings, original Stability Fees and appropriate Debt Ceilings:
+
+ * USDC-C
+ * TUSD-B
+ * PAXUSD-B
+
+The target for this executive spell is Friday, December 18th, 2020 - the approximate date at which the leading edge of vaults will reach 100% Collateralization Ratio. The Rates Working Group will run a governance poll on Monday, December 14th, 2020 to confirm the executive is still desired. This plan can also be cancelled at any point between now and December 18th, 2020 via Signal Request + Governance Poll in case the market conditions materially change.
```
Comment (LongForWisdom) [inline governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md:33]: As discussed, lets move this forward a week.

File: governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+---
+
+## Resources
```
Comment (LongForWisdom) [inline governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md:37]: Grab the latest resources from the templates. They update semi-regularly as I fix stuff.

File: governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+Please refer to the [Stablecoin Action Plan thread](https://forum.makerdao.com/t/signal-request-stablecoin-action-plan/4808) for an overview of the proposed plan.
+
+`Option 3 - Migrate Users to a new Vault Type` won the signal request. If this poll passes, the Rates Working Group will work with the Smart Contracts Team to add an executive spell which will set both the Stability Fee and Debt Ceilings to 0 on the following vault types:
+
+ * USDC-A
+ * TUSD-A
+ * PAXUSD-A
+
+ This executive will then add the following vault types with the same settings, original Stability Fees and appropriate Debt Ceilings:
```
Comment (LongForWisdom) [inline governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md:27]: I'm not sure if we want to commit to additional USDC vaults given it has the largest exposure. Maybe say that the exact new vaults added + parameters will be decided by the risk team?

**Review by LongForWisdom — CHANGES_REQUESTED** [review]

File: governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md (hunk 3 above)
Comment (hexonaut) [reply governance/polls/Rates - Stablecoin Action Plan - November 2, 2020.md:27]: Not sure I understand. Are you saying you don't think we should commit to these particular vault types, or the "same settings"?

**Review by LongForWisdom — APPROVED** [review]

**Pull request merged**